[2024-10-27T16:52:10.107+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-27T16:52:10.121+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag2.first_task manual__2024-10-27T16:52:07.189790+00:00 [queued]>
[2024-10-27T16:52:10.129+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag2.first_task manual__2024-10-27T16:52:07.189790+00:00 [queued]>
[2024-10-27T16:52:10.129+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-27T16:52:10.148+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): first_task> on 2024-10-27 16:52:07.189790+00:00
[2024-10-27T16:52:10.159+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag2', 'first_task', 'manual__2024-10-27T16:52:07.189790+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp0j3cr7f8']
[2024-10-27T16:52:10.162+0000] {standard_task_runner.py:105} INFO - Job 63: Subtask first_task
[2024-10-27T16:52:10.170+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=230) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-27T16:52:10.170+0000] {standard_task_runner.py:72} INFO - Started process 231 to run task
[2024-10-27T16:52:10.220+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag2.first_task manual__2024-10-27T16:52:07.189790+00:00 [running]> on host 2d730dc1e71d
[2024-10-27T16:52:10.355+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='johnwell' AIRFLOW_CTX_DAG_ID='our_first_dag2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-27T16:52:07.189790+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-27T16:52:07.189790+00:00'
[2024-10-27T16:52:10.355+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-27T16:52:10.371+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-27T16:52:10.371+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-10-27T16:52:10.385+0000] {subprocess.py:86} INFO - Output:
[2024-10-27T16:52:10.386+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-10-27T16:52:10.386+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-27T16:52:10.420+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-27T16:52:10.420+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag2, task_id=first_task, run_id=manual__2024-10-27T16:52:07.189790+00:00, execution_date=20241027T165207, start_date=20241027T165210, end_date=20241027T165210
[2024-10-27T16:52:10.468+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-27T16:52:10.486+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-27T16:52:10.501+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
